GitHub_Learning/Feature/Onboarding/Presentation/OnboardingPresenter.swift

```diff
@@ -12,20 +12,20 @@ import CleanPlatform
 
 protocol OnboardingPresenter: Presenter {
     func next()
-    func previous()
-    func presentingPage(at index: Int)
+    func selectedPage(at index: Int)
 }
 
 final class OnboardingPresenterImpl: BasePresenter<OnboardingView>, Listener {
     
     private let onboardingController: OnboardingController
     private var index: Int = 0 {
         didSet {
+            updateTitle()
             showPage()
         }
     }
     
-    var numberOfPages: Int {
+    private var numberOfPages: Int {
```

Comment:
Tuto property pouzivas vzdy v kontextu posledniho indexu. Muzes to tedy prejmenovat na `lastIndex` a porovnavat pak aktualni index s tim. Pak se ti nebude opakovat `numberOfPages - 1`.